docs/data/toolpad/components/button.md

```diff
@@ -0,0 +1,55 @@
+# Button
+
+<p class="description">Learn about a button component and it's usage in Toolpad.</p>
+
+Buttons allow users to take actions, make choices with a single tap. Detailed documentation of the component properties is available in the reference section for [button](/toolpad/reference/components/button/#properties).
+
+{{"demo": "ButtonBasic.js", "hideToolbar": true}}
+
+## Variations
+
+The Button component has multiple variations supported in Toolpad. Let's look at each of these below.
+
+### Variant
+
+The variant property supports three different options: contained (default), outlined and text. Contained is a high-emphasis button suited for a primary action. Outlined is for low-emphasis, secondary action. Text is used for less-pronounced actions that ensure user remains focused on the main content.
+{{"demo": "ButtonVariant.js", "hideToolbar": true}}
+
+### Color
+
+The color property has two options: primary (default) and secondary. These take input from the global theme that you set in Toolpad from the theme tab.
+{{"demo": "ButtonColor.js", "hideToolbar": true}}
+
+### Size
+
+The size property supports three options: small (default), medium and large.
+{{"demo": "ButtonSize.js", "hideToolbar": true}}
+
+### State
+
+Button supports states like loading and disabled to share the state of the component so that user can wait/act accoridngly.
+{{"demo": "ButtonState.js", "hideToolbar": true}}
+
+## Usage
+
+Button component becomes usable through an **onClick** event handler property. It is used to instruct what action to happen in the event of a click. It supports two tabs:
+
+### JS Expression
```

Comment:
This example uses a form component - that might be confusing for users, as this page is supposed to be for the `Button` component only?